### PR TITLE
Resolutions

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Documentation**

This PR resolves [MX-438]

### Description

We had 4 resolutions. 3 were some `@types/*` thing, and we had them in the devdeps anyway, so I removed them.
The last one is cheerio and I looked around and documented it in `HACKS.md`.
Lastly I also added a little comment in resolutions so we remember to write things down next time.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))

[MX-434]: https://artsyproduct.atlassian.net/browse/MX-434
[MX-438]: https://artsyproduct.atlassian.net/browse/MX-438

### CODE REVIEW

**Comment by ds300** [comment]
`tsc` is failing 🤔 looks like we need that styled-components resolution at least

**Comment by pvinis** [comment]
Mm interesting. I'll take a look.

File: HACKS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!-- Template
+
+- Title
+Tell us when we can remove this hack.
+
+Explain why the hack was added.
+
+ -->
```
Comment (ds300) [inline HACKS.md:8]: 🙌 Yes!

**Review by ds300 — APPROVED** [review]
Great stuff, thank you ❤️

**Review by ashfurrow — APPROVED** [review]
Awesome! Thank you!

**Pull request merged**